ovos_plugin_manager/language.py

```diff
@@ -135,7 +141,7 @@ def create(config=None):
         if not lang_module:
             # TODO: `language` is the only factory with this special handling
```

Comment:
do we need this special handling? it was a temporary thing while the plugins got developed in the first place

Reply:
Maybe not? There's already fallback handling if the plugin init fails; I left it for backwards-compat. last time but the exception handling should prevent anything from actually breaking